odl/tomo/data/fei_extended_header.py

```diff
@@ -0,0 +1,101 @@
+|8        |1053-1056|Float32  |DEFOCUS        |Defocus as read from the      |
+|         |         |         |               |microscope. For units see     |
+|         |         |         |               |remarks on X_STAGE.           |
++---------+---------+---------+---------------+------------------------------+
+|9        |1057-1060|Float32  |EXP_TIME       |Exposure time in seconds      |
++---------+---------+---------+---------------+------------------------------+
+|10       |1061-1064|Float32  |MEAN_INT       |Mean value of the image       |
++---------+---------+---------+---------------+------------------------------+
+|11       |1065-1068|Float32  |TILT_AXIS      |Orientation of the tilt axis  |
+|         |         |         |               |in the image in degrees.      |
+|         |         |         |               |Vertical to the top is 0      |
+|         |         |         |               |degrees, the direction of     |
+|         |         |         |               |positive rotation is          |
+|         |         |         |               |anti-clockwise.               |
++---------+---------+---------+---------------+------------------------------+
+|12       |1069-1072|Float32  |PIXEL_SIZE     |Pixel size of the images in SI|
+|         |         |         |               |units (meters)                |
++---------+---------+---------+---------------+------------------------------+
+|13       |1073-1076|Float32  |MAGNIFICATION  |Magnification used for        |
+|         |         |         |               |recording the images          |
++---------+---------+---------+---------------+------------------------------+
+|14       |1077-1080|Float32  |HT             |Value of the high tension in  |
+|         |         |         |               |SI units (volts)              |
++---------+---------+---------+---------------+------------------------------+
+|15       |1081-1084|Float32  |BINNING        |The binning of the CCD or STEM|
+|         |         |         |               |acquisition                   |
++---------+---------+---------+---------------+------------------------------+
+|16       |1085-1088|Float32  |APPLIED_DEFOCUS|The intended application      |
+|         |         |         |               |defocus in SI units (meters), |
+|         |         |         |               |as defined for example in the |
+|         |         |         |               |tomography parameters view    |
++---------+---------+---------+---------------+------------------------------+
+"""
+
+
+def print_fei_ext_header_spec():
+    """Print the specification table of an FEI extended header section.
+
+    The specification table is as follows:
+    """
+    print(MRC_FEI_EXT_HEADER_SECTION)
+
+print_fei_ext_header_spec.__doc__ += MRC_FEI_EXT_HEADER_SECTION
+
+
+MRC_FEI_SECTION_SIZE = 128
+MRC_FEI_MAX_SECTIONS = 1024
+
+ANGSTROM_IN_METERS = 1e-10
+MICRON_IN_METERS = 1e-6
```

Comment:
Well.. the initial thought was that it's somewhat optional so perhaps the functionality can be outsourced, but now all that's left is the specification and some constant definitions. I guess it can go into the main file.